plugin.yaml

```diff
@@ -2,11 +2,34 @@ name: aws-scan
 repository: github.com/aquasecurity/trivy-aws
 maintainer: aquasecurity
 summary: Trivy AWS Cloud scanning plugin
-version: "0.10.0"
+version: "0.11.0"
 usage: AWS Trivy plugin
 description: |-
   Trivy AWS plugin for the Trivy security scanner
   https://github.com/aquasecurity/trivy-aws
 platforms:
-  - uri: https://github.com/aquasecurity/trivy-aws/releases/download/v0.10.0/trivy-aws.tar.gz
-    bin: ./trivy-aws
\ No newline at end of file
+  - selector:
```

Comment:
The selectors should be updated. Now a different archive is created for each architecture.

Reply:
ah thanks, missed that. I updated it.